js/src/utilities/util.ts

```diff
@@ -0,0 +1,13 @@
+let nextId = 0;
+
+export function generateElementId() {
+	return `ax-${nextId++}`;
```

Comment:
let's use a different prefix to avoid conflict with docs-ui

Reply:
Is `ax-` in Docs-UI just an arbitrary prefix, or is there a certain naming convention that we should follow? I updated it to `bx-` , but not sure if it's a good choice.

Reply:
arbitrary yep